From 9b86710dc12f4a74474c86647011e4ee84f84c91 Mon Sep 17 00:00:00 2001
From: Sebastiano Barezzi <barezzisebastiano@gmail.com>
Date: Thu, 8 Jul 2021 15:19:41 +0200
Subject: [PATCH 084/115] drivers: power: supply: maxim: Extract S_secret from
 vayu stock kernel

`cat /sys/devices/platform/soc/soc:maxim_ds28e16/ds_S_secret`

Change-Id: I1d33a55e38b197aec5a36b2253eaece33996afbb
---
 drivers/power/supply/maxim/ds28e16.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/power/supply/maxim/ds28e16.c b/drivers/power/supply/maxim/ds28e16.c
index 2947b5b36..c3df48f55 100644
--- a/drivers/power/supply/maxim/ds28e16.c
+++ b/drivers/power/supply/maxim/ds28e16.c
@@ -55,10 +55,10 @@ unsigned char session_seed[32] = { 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
 				   0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
 				   0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
 				   0xAA, 0xAA, 0xAA, 0xAA };
-unsigned char S_secret[32] = { 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
-			       0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
-			       0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
-			       0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA };
+unsigned char S_secret[32] = { 0x0C, 0x99, 0x2B, 0xD3, 0x95, 0xDB, 0xA0, 0xB4,
+			       0xEF, 0x07, 0xB3, 0xD8, 0x75, 0xF3, 0xC7, 0xAE,
+			       0xDA, 0xC4, 0x41, 0x2F, 0x48, 0x93, 0xB5, 0xD9,
+			       0xE1, 0xE5, 0x4B, 0x20, 0x9B, 0xF3, 0x77, 0x39 };
 unsigned char challenge[32] = { 0x00 };
 int auth_ANON = 1;
 int auth_BDCONST = 1;
@@ -633,9 +633,9 @@ int DS28E16_cmd_device_disable(int op, unsigned char *password)
 /// 'Compute and Read Page Authentication' command
 ///
 /// @param[in] anon - boolean parameter
-/// @param[in] pg - Page number   2,计数�? 0,page0; 1,page1;
+/// @param[in] pg - Page number   2,计数�? 0,page0; 1,page1;
 /// @param[in] challenge
-/// @param[out] hmac   返回的计算结�?2个字�?///
+/// @param[out] hmac   返回的计算结�?2个字�?///
 /// @return
 /// DS_TRUE - command successful @n
 /// DS_FALSE - command failed
-- 
2.52.0

